spec/spec_helper.rb

```diff
@@ -1,6 +1,7 @@
 require 'rspec'
 require 'fileutils'
 require 'tempfile'
+require 'faraday'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.